tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/util/Constants.java

```diff
@@ -0,0 +1,10 @@
+
+package fi.helsinki.cs.tmc.langs.rust.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Constants {
```

Comment:
I don't really like adding any more public classes (we need to more aggressively start changing structure so that we can manage with package visible classes.)

However for now this is fine.